### PR TITLE
A few edits

### PR DESCRIPTION
**Pull request by greg-008**

### CODE REVIEW

File: profiles/bram-cohen/index.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ twitter_username: bramcohen
...
+Bram Cohen is the inventor of BitTorrent, which is the most used protocol for peer-to-peer file sharing over the internet. In 2009, BitTorrent accounted for 43% to 70% of all internet traffic. Bram is often suspected of being the elusive Satoshi Nakamoto – a claim he denies. Bram founded BitTorrent, Inc in 2004, and served as its first CEO. During his time at BitTorrent, he worked in various roles as an Engineering Manager, Product Manager, and member of the Board of Directors. He left BitTorrent in August 2017 to found Chia Network. He has served as Chairman and CTO of the Chia Network since its founding and, since June 2019, as CEO. Bram is one of the top-selling puzzle designers in the world.
```
Comment (hoffmang9) [inline profiles/bram-cohen/index.md:12]: I am being intentional with the accused language.

File: profiles/gene-hoffman/index.md [hunk 2]
```diff
@@ -9,4 +9,4 @@ twitter_username: hoffmang
...
+A serial entrepreneur, Gene Hoffman has founded no less than three companies, and oversaw their acquisitions by PGP, Inc., Vivendi-Universal, and Amdocs. From 2003 to 2016, he was co-founder, Chairman and CEO of Vindicia, a consumer subscription infrastructure company sold to Amdocs in 2016. From 2017 to 2019, he served as an advisor to the board of Chia Network, before joining the company full-time in August of 2019. He has been a board member at eight technology and energy companies, and serves on two non-profit boards. He has 23 years of experience working in high-compliance security environments, including 12 years of managing PCI and SSAE-16 compliance for the storage of 220,000,000 credit cards. Gene has built and scaled companies in enterprise software and SaaS, consumer subscriptions, cryptography, and software development, and has served as chief executive of a public company. He has raised over $215,000,000 in public and private markets, acquired four companies and sold three. In 1997, he helped put an end to US export controls on cryptography while at PGP, Inc. by personally exporting the PGP Source Code book.
```
Comment (hoffmang9) [inline profiles/gene-hoffman/index.md:12]: Heh.

File: profiles/misha-graboi/index.md [hunk 3]
```diff
@@ -9,4 +9,4 @@ twitter_username: blank
 linkedin_username: misha-graboi-cfa-frm-0ba81
 ---
 
-Prior to Chia, Misha Graboi was CEO of Delta-V Global Management, a consultancy focused on asset management firms. Prior to Delta-V, Mr. Graboi spent a decade at PAAMCO Prisma where he held several roles, most recently as CEO of PAAMCO Asia and as the sector specialist for convertible arbitrage strategies at PAAMCO. Earlier in his career, Misha was an Executive Director at Goldman Sachs, where he headed European Internet Equity Research. Misha began his career at CIBC Oppenheimer where he was a technology investment banker and Internet equity research analyst.
+Prior to joining Chia, Misha Graboi was CEO of Delta-V Global Management, a consultancy focused on asset management firms. Prior to Delta-V, Mr. Graboi spent a decade at PAAMCO Prisma where he held several roles, most recently as CEO of PAAMCO Asia and as the sector specialist for convertible arbitrage strategies at PAAMCO. Earlier in his career, Misha was an Executive Director at Goldman Sachs, where he headed European Internet Equity Research. Misha began his career at CIBC Oppenheimeri, where he was a technology investment banker and Internet equity research analyst.
```
Comment (hoffmang9) [inline profiles/misha-graboi/index.md:12]: Oppenheimer is misspelled in your proposed change

File: profiles/paul-ford/index.md [hunk 4]
```diff
@@ -9,4 +9,4 @@ twitter_username: blank
...
\ No newline at end of file
+In his role, Paul Ford is responsible for driving developer adoption of Chia, and Chia’s remarkable built-in programming language, ChiaLisp. Previous to Chia, Mr. Ford acted as CMO of Storj, a crypto data storage company, where he was responsible for both product and marketing, helping drive its market value to almost $1BN USD. Before Storj, he was VP of SendGrid ($1.5BN IPO, 2017) and then Twilio SendGrid ($3BN Acquisition, 2018). Paul also saw great success as VP of API-based datacenter company SoftLayer in Dallas, TX ($2.5BN Acquisition by IBM). Other engagements include Corporate Developement at Rackspace (RAX), and 2X as a VC-funded startup co-founder.
```
Comment (hoffmang9) [inline profiles/paul-ford/index.md:12]: It is "Chialisp" "acted as" feels off but agree that something better can go there.

File: profiles/paul-hainsworth/index.md [hunk 5]
```diff
@@ -9,4 +9,4 @@ twitter_username: blank
 linkedin_username: blank
 ---
 
-Prior to joining Chia, Paul Hainsworth was CEO of Open Garden, which created peer to peer (P2P) messaging app FireChat, and in 2018 launched a blockchain-based P2P WiFi internet sharing network protocol backed by August Capital, Verizon Ventures, and more. Earlier in his career, he ran product management for BlackBerry’s app store and developer ecosystem, Sprint’s $1B annual revenue prepaid value-added services business, and played various product management roles at Virgin Mobile USA and Helio.
\ No newline at end of file
+Prior to joining Chia, Paul Hainsworth was CEO of Open Garden, which created the peer-to-peer (P2P) messaging app FireChat, and in 2018 launched a blockchain-based solution for sharing one's internet subscription (Airbnb for internet access). While at Open Garden, Paul oversaw funding rounds by August Capital, Verizon Ventures, among other venture capital firms. Earlier in his career, he ran product management for BlackBerry’s app store and developer ecosystem, Sprint’s $1B annual revenue prepaid value-added services business, and played various product management roles at Virgin Mobile USA and Helio.
```
Comment (hoffmang9) [inline profiles/paul-hainsworth/index.md:12]: Make sure @paulhainsworth-chia is ok with these.

File: profiles/russell-saito/index.md [hunk 6]
```diff
@@ -9,6 +9,6 @@ twitter_username: blank
...
+Russell leads Chia Network's initiatives in Japan, and is a multi-cultural business executive with 15+ years of professional experience in Asia. As an expert in financial technology and corporate innovation, he also serves as an outside advisor to Nikkei Inc., Japan's largest Business Newspaper. Before joining Chia, he was Vice-President at Trust Capital (formerly Mitsui Sumitomo Trust Capital), one of the leading mezzanine financing fund in Japan. Russell has held leadership positions with global technology corporations, and has extensive cross-sector experience working with governments and nonprofits.
```
Comment (hoffmang9) [inline profiles/russell-saito/index.md:12]: Note that funds would need to be capitalized.

File: profiles/steve-booth/index.md [hunk 7]
```diff
@@ -9,4 +9,4 @@ twitter_username: blank
...
\ No newline at end of file
+Steve Booth is a seasoned sales executive with over 30 years of enterprise solution sales and sales management experience. Steve is passionate about the opportunities for distributed ledger technology to truly change the world for the better, and is focused on Chia blockchain adoption across the globe.  Prior to joining Chia, he served as vice president of sales at Vindicia. Before that, he was vice president of sales and business development at Proclivity Systems. Steve has also held senior sales management positions at Aria Systems, QlikTech, Responsys, Portal Software, Oracle, and ADP. He holds a BS in marketing and finance from LaSalle University.
```
Comment (hoffmang9) [inline profiles/steve-booth/index.md:12]: I loathe DLT. It's either blockchain or secure distributed database.

File: profiles/vishal-kapoor/index.md [hunk 8]
```diff
@@ -9,4 +9,4 @@ twitter_username: blank
...
\ No newline at end of file
+Prior to Chia, Vishal Kapoor served as the CEO of Paradata, a machine learning-based RPA engine for analyzing and adjudicating supply chain components data. He is also a partner at Three Legged Stool - a company that creates software solutions to tackle the challenges posed by the COVID crisis and other related challenges around education, healthcare and workforce segments. Earlier in his career, Vishal worked in engineering, sales and product management at Alcatel, Silicon Access Networks, Verisity, and Cadence. He started his career engaged in the discovery of the Higgs Boson, developing big data and tiered data acquisition systems at the Superconducting Super Collider for the U.S. Department of Energy.
```
Comment (hoffmang9) [inline profiles/vishal-kapoor/index.md:12]: Again, probably needs a check with VIshal but I'm fine with it.

**Review by hoffmang9 — CHANGES_REQUESTED** [review]

File: profiles/steve-booth/index.md (hunk 7 above)
Comment (greg-008) [reply profiles/steve-booth/index.md:12]: Point taken :-) I don't like it either but it's used in many business publications.

**Pull request closed without merging**